### PR TITLE
Setting this project to be uploaded as gradle artifact

### PR DESCRIPTION
**Pull request by raulccabreu**
to https://bintray.com maven repository and update example to import this project as a remote dependency inside personal maven cloud repo

### CODE REVIEW

File: client/android/example/app/build.gradle [hunk 1]
```diff
@@ -22,6 +22,6 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile project(path: ':lib')
+    compile 'br.com.catbag.switchboard:android-client:1.0.0'
```
Comment (niltonvasques) [inline client/android/example/app/build.gradle:25]: :+1:

**Comment by raulccabreu** [comment]
This PR was based on bintray sample: https://github.com/bintray/bintray-examples/tree/master/gradle-bintray-plugin-examples/android-maven-example

File: client/android/project/build.gradle [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+group = release_group
+version = release_version
+Properties properties = new Properties()
+properties.load(project.rootProject.file('local.properties').newDataInputStream())
+bintray {
+    //Put this two on local.properties
+    //bintray.user=<your user>
+    //bintray.apikey=<your api key>
+    user = properties.getProperty("bintray.user")
+    key = properties.getProperty("bintray.apikey")
```
Comment (niltonvasques) [inline client/android/project/build.gradle:61]: Its required for all users that want download this module, have an account in bintray?

**Comment by niltonvasques** [comment]
:+1: Good job mate!

File: client/android/project/build.gradle (hunk 2 above)
Comment (raulccabreu) [inline client/android/project/build.gradle:61]: No, this is the upload code, to upload a artifact you have to signup and setup a repo. But to download, its only necessary to point a specifc artifact repo and compile the artifact.

**Pull request merged**